FBTweak/_FBEditableTweakTableViewCell.m

```diff
@@ -315,7 +323,12 @@ - (void)_updateValue:(FBTweakValue)value primary:(BOOL)primary write:(BOOL)write
     if (primary) {
       _textField.text = value;
     }
-  } else if (_mode == _FBTweakTableViewCellModeInteger) {
+  } else if(_mode == _FBTweakTableViewCellModeDate){
+    if (primary) {
+      _textField.text = [((NSDate *) value) description];
```

Comment:
remove space before `value`

Reply:
I'm not sure if `description` is the way to go here. Take a look at `NSDateFormatter` and do something more sensible.